meilisearch-http/src/index/mod.rs

```diff
@@ -174,7 +174,7 @@ impl Index {
         txn: &heed::RoTxn,
         attributes_to_retrieve: &Option<Vec<S>>,
         fields_ids_map: &milli::FieldsIdsMap,
-    ) -> Result<Vec<u8>> {
+    ) -> Result<Vec<u16>> {
```

Comment:
lets use the type alias `FieldId` here instead
```suggestion
    ) -> Result<Vec<FieldId>> {
```